app/components-react/root/NotificationsArea.m.less

```diff
@@ -69,6 +75,9 @@
   cursor: pointer;
   white-space: nowrap;
   margin-right: 10px;
+  height: 100%;
+  display: flex;
+  align-items: center;
```

Comment:
this looks kinda weird will small notifications (see screenshot)
![image](https://user-images.githubusercontent.com/20602520/201211676-f2973fcc-967a-439f-bea3-388e37f44468.png)